pkg/networkservice/up/server.go

```diff
@@ -59,13 +61,15 @@ func (u *upServer) Request(ctx context.Context, request *networkservice.NetworkS
 		if closeErr := u.closeOnFailure(postponeCtxFunc, conn); closeErr != nil {
 			err = errors.Wrapf(err, "connection closed with error: %s", closeErr.Error())
 		}
+		log.FromContext(ctx).Errorf("SOME UGLY ERROR: %v", err.Error())
```

Comment:
I like idea for adding more logging, could you correct the message?

Reply:
I suppose that we will see this error in logs, because we are logging every returned error from every chain element. Probably we don't actually need any logs here.

Reply:
Removed excessive logs. It was needed for debugging puproses.